pkg/modelconv/modelconv.go

```diff
@@ -3,3 +3,10 @@ package modelconv
 import "github.com/iver-wharf/wharf-core/pkg/logger"
 
 var log = logger.NewScoped("WHARF")
+
+func fallbackString(value, fallback string) string {
```

Comment:
If there's a C# params style parameter in Go it would be cool to be able to send in multiple fallbacks and return the first one with content by looping through them.

Reply:
Yes there is! As nice use case as that is though, I would like to wait with implementing that until we need it.

Reply:
I went with variadic params anyway. See the implementation in `internal/or/or.go`